src/rpcrawtransaction.cpp

```diff
@@ -259,7 +259,7 @@ UniValue listunspent(const UniValue& params, bool fHelp)
     vector<COutput> vecOutputs;
     assert(pwalletMain != NULL);
     LOCK2(cs_main, pwalletMain->cs_wallet);
-    pwalletMain->AvailableCoins(vecOutputs, false);
+    pwalletMain->AvailableCoins(vecOutputs, false, NULL, false, ALL_COINS, false, true);
```

Comment:
People will kill you for that. Remember this is the output of an RPC command, so the whole world might have to change their processing chain, or in the worst case (which is in my experience the 100%  case for changes like this) won't notice and process wrong numbers.
Please add an additional optional(!) parameter to the `listunspent` call to keep the existing behavior valid, e.g.

1.  `listunspent`: same behavior as before
2.  `listunspent watchonly=no`: same behavior as before
3.  `listunspent watchonly=yes`: list ONLY watchonly coins
4.  `listunspent watchonly=all`: list both

I think this will call all possible use-cases.